libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Identity/SFIdentityData.h

```diff
@@ -220,22 +220,22 @@
 /**
  * An optional dictionary of custom attributes defined on the Connected App.
  */
-@property (strong, nonatomic, readonly) NSDictionary *customAttributes;
+@property (nullable, strong, nonatomic, readonly) NSDictionary *customAttributes;
 
 /**
  * An optional dictionary of custom permissions defined on the Connected App.
  */
-@property (strong, nonatomic, readonly) NSDictionary *customPermissions;
+@property (nullable, strong, nonatomic, readonly) NSDictionary *customPermissions;
 
 /**
  * The date this record was last modified.
  */
-@property (strong, nonatomic, readonly) NSDate *lastModifiedDate;
+@property (nonnull, strong, nonatomic, readonly) NSDate *lastModifiedDate;
 
 /**
  * Designated initializer for creating an instance of the SFIdentityData object.
  * @param jsonDict The JSON dictionary containing the user data.
  */
-- (id)initWithJsonDict:(NSDictionary *)jsonDict;
+- (nonnull instancetype)initWithJsonDict:(nonnull NSDictionary *)jsonDict;
```

Comment:
changed to use the new style of declare init (i.e. use instancetype instead of id)